src/services

```diff
@@ -35,7 +35,7 @@ p({"class":"foam.nanos.boot.NSpec", "name":"email",                       "lazy"
 p({"class":"foam.nanos.boot.NSpec", "name":"emailTemplateDAO",            "lazy":true,  "serve":true,  "serviceScript":"return new foam.dao.PMDAO(new foam.dao.SequenceNumberDAO(new foam.dao.JDAO(x, foam.nanos.notification.email.EmailTemplate.getOwnClassInfo(), \"emailTemplates\")));","client":"{\"of\":\"foam.nanos.notification.email.EmailTemplate\"}" })
 p({"class":"foam.nanos.boot.NSpec", "name":"authy",                       "lazy":true,  "serve":false, "serviceClass":"foam.nanos.auth.twofactor.authy.AuthyService","service":{"class":"foam.nanos.auth.twofactor.authy.AuthyService","debug":true,"apiHost":"https://api.authy.com","apiKey":""}})
 p({"class":"foam.nanos.boot.NSpec", "name":"blobStore",                   "lazy":true,  "serve":false, "serviceClass":"foam.blob.BlobStore"})
-p({"class":"foam.nanos.boot.NSpec", "name":"httpBlobService",             "lazy":true,  "serve":false, "authenticate": false, "serviceScript":"return new foam.nanos.blob.HttpBlobService(x, x.get(\"blobStore\"));"})
```

Comment:
I think fileService should be its own service rather than repurposing httpBlobService.

Reply:
Okay sure